decide/visualizer/static/liveresults/liveresults.js

```diff
@@ -0,0 +1,90 @@
+(() => {
+    class VotingCount {
+        constructor(app, options, fetchInterval = 2500) {
+            this.totalVotes = 0;
+            this.app = app;
+            this.fetchInterval = fetchInterval;
+            this.options = options;
+            this.intervalFetch();
+        }
+
+        /*
+         * Descripción: Ejecuta de manera periódica según el parámetro fetchInterval una función de la app de Vue definida para recoger el recuento de votos de la base de datos
+         * Entrada: Ninguna
+         * Salida: Ninguna
+         */
+        intervalFetch() {
+            setInterval(() => {
+                console.log(`Refrescando resultados de la votación: ${voting.id}`);
+        
+                this.app.getVotingCount().then(data => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/Penyagolosa-3/Decide/pullRequest?prid=8608522)
```suggestion
                this.app.getVotingCount().then((data) => {
```